cli/cmd/project/project.go

```diff
@@ -40,12 +40,14 @@ func toRow(o *adminv1.Project) *project {
 		Public:    o.Public,
 		GithubURL: o.GithubUrl,
 		CreatedAt: o.CreatedOn.AsTime().String(),
+		Envs:      o.Envs,
 	}
 }
 
 type project struct {
-	Name      string `header:"name" json:"name"`
-	Public    bool   `header:"public" json:"public"`
-	GithubURL string `header:"github" json:"github"`
-	CreatedAt string `header:"created_at,timestamp(ms|utc|human)" json:"created_at"`
+	Name      string            `header:"name" json:"name"`
+	Public    bool              `header:"public" json:"public"`
+	GithubURL string            `header:"github" json:"github"`
+	CreatedAt string            `header:"created_at,timestamp(ms|utc|human)" json:"created_at"`
+	Envs      map[string]string `header:"variables" json:"variables"`
```

Comment:
This gets printed as a table, so if env contains just e.g. an S3 access token, this can mess up the formatting. Let's remove from here. We could add a `rill project env show` command to list out the env specifically? (In `XXX=YYY` format with newlines for each value)